OptionSelector/DetailCategoryTableViewController.h

```diff
@@ -0,0 +1,16 @@
+//
+//  DetailCategoryTableViewController.h
+//  OptionSelector
+//
+//  Created by Jason Wang on 8/15/15.
+//  Copyright (c) 2015 Mike Kavouras. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+#import "CQCategory.h"
```

Comment:
Style: It's better to keep `#import` directives in your .m files. You can replace this with a forward declaration (i.e. `@class CQCategory;`) in your .h file.